selectrum.el

```diff
@@ -337,6 +337,14 @@ setting."
                        (string :tag "Indicator string")
                        (face :tag "Indicator face"))))
 
+(defcustom selectrum-highlight-entire-line nil
+  "Whether to highlight the entire line of the current candidate.
+
+The default is to only highlight the displayed candidate text,
+though if the candidate has text displayed at the right margin,
+then the entire line will be highlighted anyway."
```

Comment:
Could we shorten this to "The default is to only highlight the displayed text"? That would be a bit easier to read/understand.

Reply:
It is changed.

Reply:
Thanks, I meant leaving off the ",though if..." part, too. I don't think it's needed because " The default is to only highlight the displayed text" should be sufficient. The `though if...` only describes a detail following from this statement, what do you think?

Reply:
I removed it.

Reply:
Nice, thanks!